### DISCUSSION TITLE
一款产品的从0到1之旅 · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#142] [Announcements]
# 一款产品的从0到1之旅 · BMPI

记录交易日记App开发的过程，包括移动端App以及服务端的全栈开发过程，还有一些关于产品运营及营销知识的介绍

https://www.bmpi.dev/dev/zero-to-one/

**Comment by zsyoung** [comment]
太强了！

**Comment by zengyishou** [comment]
没想到是16年的文章，里面所表达的“全栈”思想，我在今年1月份的时候才悟到，自己的顿悟原来只是他人的基础。佩服，顺着myGPTReader
